packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4924,7 +4927,7 @@ class RichText extends LeafRenderObjectWidget {
   ///
   /// The [textDirection], if null, defaults to the ambient [Directionality],
   /// which in that case must not be null.
-  const RichText({
+  RichText({
```

Comment:
This is a breaking change

Reply:
Yeah, this was announced already.

Reply:
Should be able to replace instances of RichText with Text.rich migrate.